packages/sdk-snippets/src/index.ts

```diff
@@ -49,7 +49,7 @@ export class MetricsSDKSnippet {
     const { convert } = target.services[snippetType][clientId || target.info.default];
 
     // Reduce our variables into a set keyed by their type.
-    const { server, security } = this.variables.reduce(
+    const { server, security } = this.variables.reduce<any>(
```

Comment:
could we make this a little narrower?

```suggestion
    const { server, security } = this.variables.reduce<{ server: any[]; security: any[] }>(
```

Reply:
i tried and ran into a bunch of issues with it thinking that `server` and `security` are `never[]`. this is me throwing my hands up